Index: src/OopsBannerApp.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package src;\r\n\r\n/**\r\n * * OOPS Banner App UC1 – OOPS Banner Display Application\r\n * <p>\r\n * This class demonstrates a simple Java application\r\n * that displays the Object Oriented Programming System (OOPS)\r\n * acronym to the console.\r\n *\r\n * @author User\r\n * @version 1.0\r\n */\r\npublic class OopsBannerApp {\r\n    public static void main(String[] args) {\r\n        System.out.println(\"OOPS\");\r\n    }\r\n}\r\n
===================================================================
diff --git a/src/OopsBannerApp.java b/src/OopsBannerApp.java
--- a/src/OopsBannerApp.java	(revision 05c2ad07fdfe380c4a79ed294d3705486741770c)
+++ b/src/OopsBannerApp.java	(date 1769668500229)
@@ -10,6 +10,7 @@
  * @author User
  * @version 1.0
  */
+
 public class OopsBannerApp {
     public static void main(String[] args) {
         System.out.println("OOPS");
